hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1507,6 +1509,13 @@ public void stop() {
     }
 
     scmSafeModeManager.stop();
+
+    try {
+      LOG.info("Stopping SCM HA services.");
+      scmHAManager.shutdown();
+    } catch (Exception ex) {
+      LOG.error("SCM HA Manager stop failed", ex);
+    }
```

Comment:
Can you please explain this change in order of statements?

Reply:
`scmHAManager.shutdown()` will call `stateMachine#close`.  if we want `stateMachine#close` to terminate the running process, i think it is better to stop all other servers before the process is terminated